front/src/views/BoxCreate/BoxCreateView.tsx

```diff
@@ -222,27 +240,36 @@ function BoxCreateView() {
   };
 
   // Handle Loading State
-  if (qrCodeExists.loading || allFormOptions.loading || createBoxMutationState.loading) {
+  if (qrCodeExists.loading || allFormOptions.loading || createBoxMutationState.loading)
     return <APILoadingIndicator />;
-  }
 
   // TODO: handle errors not with empty div, but forward or roll data back in the view
   if (
     !qrCodeExists.data?.qrExists ||
     qrCodeExists.error ||
     allLocations === undefined ||
```

Comment:
replace the case with a error state of allFormOptions or !allFormOption.data.base is undefined.

Reply:
Should this be for the whole if statement?

Reply:
remove the cases for `allLocations === undefined` and `allProducts === undefined`